MeetupPlatformApi/appsettings.json

```diff
@@ -4,5 +4,9 @@
       "Default": "Information",
       "Microsoft.AspNetCore": "Warning"
     }
+  },
+
+  "ConnectionStrings": {
+    "DbConnectionString": "Host=<host>;Port=<port>;Database=<db-name>;Username=<userName>;Password=<password>"
```

Comment:
Consider renaming configuration option to "PostgreSQL"

Reply:
Use only one convention for template naming: use either `kebab-case` like in `<db-name>` or `camelCase` like in `<userName>`